app/components/table-row.js

```diff
@@ -128,18 +129,16 @@ const DeleteButton = ({ ...props }) => (
   />
 )
 
-const TitleField = ({ dat }) => (
-  <div>
-    <h2 className='f6 f5-l normal truncate pr3'>
-      {dat.metadata.title || `#${dat.key}`}
-    </h2>
-  </div>
-)
-
 const Row = ({ dat, shareDat, onDeleteDat, inspectDat, onTogglePause }) => (
   <Tr
     onClick={ev => {
-      if (ev.target.tagName === 'SVG' || ev.target.tagName === 'use') return
+      if (
```

Comment:
can be refactored using an array: `if (['SVG', 'use', 'INPUT'].includes(ev.target.tagName))`